flake.nix

```
@@ -19,6 +19,8 @@
     srvos.url = "github:numtide/srvos";
     # actually not used when using the modules but than nothing ever will try to fetch this nixpkgs variant
     srvos.inputs.nixpkgs.follows = "nixpkgs";
+    srvos.inputs.flake-parts.follows = "flake-parts";
+    srvos.inputs.flake-parts.inputs.nixpkgs-lib.follows = "nixpkgs";
```

```suggestion
    srvos.inputs.flake-parts.inputs.nixpkgs-lib.follows = "nixpkgs";
```

This override usually does not work. But I opened this one here: https://github.com/numtide/srvos/pull/76

---

Thanks, updated.